### PR TITLE
Update changelog and version for v0.2.0

### LINKED ISSUE #207: release v0.2.0
Release Twoliter version v0.2.0.

- [ ] #205
- [ ] #198 
- [ ] Changelog
- [ ] Test in Bottlerocket (with necessary changes and PR)
- [ ] Test with another OOTB project (with necessary changes and PR)

### PR DESCRIPTION
**Pull request by sumukhballal**
**Issue number:**

Closes #207

**Description of changes:**

Updates twoliter to v0.2.0.

**Testing done:**

- [x] Build a bottlerocket variant and it works as is.

**Terms of contribution:**

By submitting this pull request, I agree that this contribution is dual-licensed under the terms of both the Apache License, version 2.0, and the MIT license.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,50 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.2.0] - 2024-05-29
+
+### Added
+
+- Added a dependency graph algorithm for package and kit builds ([#198])
+- Add support to repack a variant ([#214], [#211], [#217], [#219], [#221], [#222], [#228], [#231], [#235], [#243])
+- Added a Buildsys build kit command ([#227])
```
Comment (webern) [inline CHANGELOG.md]:
```suggestion
- Added the buildsys build-kit command to create kits ([#198], [#227])
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -7,6 +7,50 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.2.0] - 2024-05-29
+
+### Added
+
+- Added a dependency graph algorithm for package and kit builds ([#198])
```
Comment (webern) [inline CHANGELOG.md]:
```suggestion
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -7,6 +7,50 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
+[#228]: https://github.com/bottlerocket-os/twoliter/pull/228
+[#231]: https://github.com/bottlerocket-os/twoliter/pull/231
+[#232]: https://github.com/bottlerocket-os/twoliter/pull/232
+[#243]: https://github.com/bottlerocket-os/twoliter/pull/243
+
+### Changed
+
+- Regenerate kernel module if possible in rpm2img ([#205])
+- Changes and fixes to better support kits ([#210], [#216], [#218], [#223], [#224], [#226], [#234], [#238])
+- Remove variant sensitivity from buildsys ([#220])
```
Comment (webern) [inline CHANGELOG.md]:
```suggestion
- Deprecate variant sensitivity for packages in buildsys ([#220])
```

**Review by webern — COMMENTED** [review]
You will need to run `cargo check` or `cargo build` to update `Cargo.lock` in the same commit where you bump the version in Twoliter's Cargo.toml. CI checks fail if the Cargo.lock file is not up-to-date.

You should also do a commit where you update dependencies with `cargo update`.

Thanks!

**Comment by sumukhballal** [comment]
Updated changelog to address @webern's concern.

**Review by webern — APPROVED** [review]

File: CHANGELOG.md [hunk 4]
```diff
@@ -7,6 +7,50 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.2.0] - 2024-05-29
+
+### Added
+
+- Add support to repack a variant ([#214], [#211], [#217], [#219], [#221], [#222], [#228], [#231], [#235], [#236], [#243])
```
Comment (jpculp) [inline CHANGELOG.md]: #236 is a tangential, but standalone feature. Let's add a new line for "Add support to fetch a variant"

**Review by jpculp — CHANGES_REQUESTED** [review]

**Comment by sumukhballal** [comment]
Adding change to address @jpculp 's comment.

**Review by jpculp — APPROVED** [review]

**Pull request merged**